app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -29,6 +29,7 @@ def show
             param :name, String, :required => true
             param :value, String, :required => true
           end
+          param :effective_user, String, :required => false
```

Comment:
I went for this path just because it should just work in hammer then - with nested hash, I guess there will be some additional work needed on the hammer side. But I agree, since it's provider specific , it would be worth the effort. What about nesting this into under 'ssh' key?